### PR TITLE
Add erfinv API

### PR DESCRIPTION
**Pull request by Avin0323**
### PR types

New features

### PR changes

APIs

### Describe

新增`erfinv` API。

英文文档：

image

### CODE REVIEW

File: paddle/fluid/operators/erfinv_op.cc [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+class ErfinvOp : public framework::OperatorWithKernel {
+ public:
+  using framework::OperatorWithKernel::OperatorWithKernel;
+
+  void InferShape(framework::InferShapeContext* ctx) const override {
+    OP_INOUT_CHECK(ctx->HasInput("X"), "Input", "X", "erfinv");
+    OP_INOUT_CHECK(ctx->HasOutput("Out"), "Output", "Out", "erfinv");
+
+    ctx->SetOutputDim("Out", ctx->GetInputDim("X"));
+    ctx->ShareLoD("X", /*->*/ "Out");
```
Comment (zhhsplendid) [inline paddle/fluid/operators/erfinv_op.cc:29]: What's this comment `/*->*/`? Does it mean that "X" is also the "Out"?

**Review by zhhsplendid — APPROVED** [review]
LGTM

File: paddle/fluid/operators/erfinv_op.cc (hunk 1 above)
Comment (zhhsplendid) [reply paddle/fluid/operators/erfinv_op.cc:29]: A lot of old code commented like this. I marked as resolved and we would make it clear in the future.

**Review by Superjomn — APPROVED** [review]
LGTM

File: python/paddle/__init__.py [hunk 2]
```diff
@@ -234,6 +234,7 @@
 from .tensor.math import asinh  # noqa: F401
 from .tensor.math import atanh  # noqa: F401
 from .tensor.math import lerp  # noqa: F401
+from .tensor.math import erfinv  # noqa: F401
```
Comment (dingjiaweiww) [inline python/paddle/__init__.py:237]: erfinv_ 没有import 和 加入__all__ 列表

**Review by dingjiaweiww — APPROVED** [review]

**Pull request merged**